profiles/tests/test_crud_profile.py

```diff
@@ -0,0 +1,302 @@
+from rest_framework.test import APITestCase, APIClient
+from authentication.models import CustomUser
+from profiles.models import Profile, Category, Activity
+from utils.dump_response import dump # noqa
+import io
+from PIL import Image
+import os
+
+
+
+class TestProfileDetailAPIView(APITestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.test_person = CustomUser.objects.create_user(
+            person_email="test@test.com",
+            password="Testing01",
+            person_name="test",
+            person_surname="test",
+            is_active=True
+        )
+        cls.test_person2 = CustomUser.objects.create_user(
+            person_email="test2@test.com",
+            password="Testing01",
+            person_name="test",
+            person_surname="test",
+            is_active=True
+        )
+
+        cls.test_person3 = CustomUser.objects.create_user(
+            person_email="test3@test.com",
+            password="Testing01",
+            person_name="test",
+            person_surname="test",
+            is_active=True
+        )
+
+        cls.test_profile = Profile.objects.create(
+            person=cls.test_person,
+            comp_is_startup=True,
+            comp_registered=False
+        )
+        cls.test_profile2 = Profile.objects.create(
+            person=cls.test_person2,
+            comp_registered=True,
+            comp_is_startup=False
+        )
+
+        Category.objects.create(category_id=1,
```

Comment:
"Producer" is a type of Activitty, not Category, as I understand. Better change that to `cheese` or something.

Reply:
Thank you. Done